css/styles.css

```diff
@@ -497,10 +497,6 @@ section#developers .section-header {
 
```

Comment:
> to work with symlinks (if jekyll and friends like that)?

I think this won't work, at least I wasn't very successful when I experimented with such a solution.
What we will indeed need sooner or later is a better manageable copy mechanism, doing it in the pom.xml as it is now will definitely become too complex at some point in time.

> yes I think so, we should copy as much as possible

Feel free to adapt the copy mechanism accordingly.